datasets/interpress_news_category_tr_lite/README.md

```diff
@@ -0,0 +1,139 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- tr
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 100k<n<1M
+source_datasets:
+- original
+task_categories:
+- text_classification
+task_ids:
+- text_classification-other-news-category-classification
+---
+
+# Dataset Card for Interpress Turkish News Category Dataset (270K - Lite Version)
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Discussion of Social Impact and Biases](#discussion-of-social-impact-and-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** [Interpress](https://www.interpress.com/)
+- **Point of Contact:** [Yavuz Komecoglu](mailto:yavuz.komecoglu@kodiks.com)
+
+### Dataset Summary
+
+Turkish News Category Dataset (270K - Lite Version) is a Turkish news data set consisting of 273601 news in 10 categories ("kültürsanat", "ekonomi", "siyaset", "eğitim", "dünya", "spor", "teknoloji", "magazin", "sağlık", "gündem"), compiled from printed media and news websites between 2010 and 2017 by the Interpress (https://www.interpress.com/) media monitoring company. **It has been rearranged as easily separable and with fewer classes.**
+
+### Languages
+
+The dataset is based on Turkish.
+
+## Dataset Structure
+
+### Data Instances
+
+A text classification dataset with 10 different news category. 
+
+### Data Fields
+
+[More Information Needed]
```

Comment:
Can you add a brief description of each field `content` and `category` here please ?